src/metalnx-web/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "buildx": "npm -v",
-    "build": "vue-cli-service build --mode production",
+    "build": "vue-cli-service build --mode development",
```

Comment:
is this now correct?   should we put it back to production before merging to master?